### ISSUE TITLE
New user import;  import user into slaves failing

### ISSUE THREAD

**Issue by pronkster** [fanno/org.jfusion.phpbb31#2]
New user import;  - import user into slaves

Could not create the user: Unknown column 'user_pass_convert' in 'field list' SQL=INSERT INTO `phpbb_users`

**Comment by fanno** [comment]
the user_pass_convert column should also have been removed is not in user.php anymore

https://github.com/fanno/org.jfusion.phpbb31/blob/master/user.php (search for: user_pass_convert)

not sure why you are getting this error.

**Comment by pronkster** [comment]
I'll have another try again soon, I may have used the wrong files thinking about it now.

**Comment by fanno** [comment]
you have to checkout this repo and then zip it. and install it from there..

If a plugin by name phpbb31 is already installed it need to be removed first.

You can also download the arctive from github and browse in to the first folder. (where all the files are and zip the files there and install)

**Comment by pronkster** [comment]
I have downloaded the plugin and double checked and it is indeed now working. All users are successfully imported.

**Comment by fanno** [comment]
I was the weekend from trying to install it using the Joomla installer? And not via jfusion cpanel? 

I think that could cause the error you talked about. Earlier!!

**Comment by pronkster** [comment]
Yes I realised that after I submitted and then deleted the comment. You must have received the email. :)

All good, it working again. Great job.

**Comment by pronkster** [comment]
Hi Fanno,

I have installed the latest plugins though JFusion (version 2.1.0-3 Rev) as I thought all the issues with PHPBB3.x were fixed but it appears this one still exists. I looked at the source code of the plugin and user_pass_convert is still required. Can you please help?

**Comment by fanno** [comment]
are you sure you install the correct version ?

**Comment by pronkster** [comment]
I copied and pasted the version number i send you of my console. I have also just done a plugin version upgrade a few days ago.

I fixed it quickly by downloading the github version but that one doesn't get updates so that's why I went back.

Sent from my HTC

----- Reply message -----

**Comment by fanno** [comment]
The jfusion version number don't show what plugin you are using it only show what version of jfusion it self.
